scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/JobEmailNotification.java

```diff
@@ -107,6 +107,12 @@ public JobEmailNotification(JobState js, NotificationData<JobInfo> notification,
 
     public boolean doCheckAndSend(boolean withAttachment)
             throws JobEmailNotificationException, IOException, UnknownJobException, PermissionException {
+        String eventFinishedWithErrorsName = SchedulerEvent.JOB_RUNNING_TO_FINISHED_WITH_ERRORS.name().toLowerCase();
+        String eventFinishedWithErrorsMethod = SchedulerEvent.JOB_RUNNING_TO_FINISHED_WITH_ERRORS.toString()
+                                                                                                 .toLowerCase();
+        String eventFinishedName = SchedulerEvent.JOB_RUNNING_TO_FINISHED.name().toLowerCase();
+        String eventFinishedMethod = SchedulerEvent.JOB_RUNNING_TO_FINISHED.toString().toLowerCase();
+
         String jobStatus = jobState.getGenericInformation().get(GENERIC_INFORMATION_KEY_NOTIFICATION_EVENT);
         List<String> jobStatusList = new ArrayList<>();
```

Comment:
I think that you can specify the initial size of the ArrayList which is 10. It is more efficient as explained here https://stackoverflow.com/questions/15430247/why-start-an-arraylist-with-an-initial-capacity

Reply:
True but that `jobStatusList ` has a static content defined as a stream of static notification events so its size won't grow up. Though, It was like that before this PR :p